share/spice/git_book_status/content.handlebars

```diff
@@ -0,0 +1,16 @@
+<h3 class="gbstatus--header text--primary">
+    {{! Check if the status is either good, minor or major }}
+    {{#GitbookStatus_indicator status "none"}}
+        The status of GitBook is <span class="gbstatus--good">good</span>.
+    {{else}}
+        {{#GitbookStatus_indicator status "minor"}}
+            GitBook status is <span class="gbstatus--minor">{{status}}</span>.
+        {{else}}
+            GitBook status is <span class="gbstatus--major">{{status}}</span>.
+        {{/GitbookStatus_indicator}}
+    {{/GitbookStatus_indicator}}
+    
+    {{#GitbookStatus_ifNotEmpty description}}
+        <h6 class="gbstatus--message text--secondary">{{description}}</h6>
```

Comment:
With the `text` template we can define a `subtitle` element and then we won't need to modify it with any CSS